src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2439,6 +2439,7 @@ because xbuild doesn't support framework reference assemblies.
 		ToolExe="$(JarsignerToolExe)"
 		TimestampAuthorityUrl="$(JarsignerTimestampAuthorityUrl)"
 		TimestampAuthorityCertificateAlias="$(JarsignerTimestampAuthorityCertificateAlias)"
+		SigningAlgorithm="$(_ApkSigningAlgorithm)"
```

Comment:
we should use `ApkSigningAlgorithm` rather than `_ApkSigningAlgorithm`. In msbuild the convention is `_` prefixed properties are private and should not be used by the end user. Ones without are public. So we should remove the `_` since we expect the user to override this.